src/OSPSuite.UI/Controls/UxRepositoryItemPopupContainerEditOKButton.cs

```diff
@@ -0,0 +1,123 @@
+﻿using System.ComponentModel;
+using DevExpress.XtraEditors;
+using DevExpress.XtraEditors.Drawing;
+using DevExpress.XtraEditors.Popup;
+using DevExpress.XtraEditors.Registrator;
+using DevExpress.XtraEditors.Repository;
+using DevExpress.XtraEditors.ViewInfo;
+
+namespace OSPSuite.UI.Controls
+{
+   //adapted from
+   //https://supportcenter.devexpress.com/ticket/details/e3666/how-to-display-the-default-ok-button-in-the-popupcontaineredit-s-popup-window#PopupContainerEditOKButton.cs
+
+   public class UxRepositoryItemPopupContainerEditOKButton : RepositoryItemPopupContainerEdit
```

Comment:
@abdelr @georgeDaskalakis @Yuri05 Can't believe there is so much code to write. But it looks with this code like this
![image](https://user-images.githubusercontent.com/1041237/152789585-d4cf4eeb-f8b4-4f54-9f60-1907c425564d.png)

Reply:
Very nice! Will make some users very happy :)

Reply:
Yep. We should use this guy for all instances of REposityPopupContainer